src/main/java/io/nats/client/api/ConsumerInfo.java

```diff
@@ -46,13 +46,17 @@ public ConsumerInfo(Message msg) {
 
     public ConsumerInfo(String json) {
         super(json);
-        stream = JsonUtils.readString(json, STREAM_NAME_RE);
-        name = JsonUtils.readString(json, NAME_RE);
-        created = JsonUtils.readDate(json, CREATED_RE);
 
         String jsonObject = JsonUtils.getJsonObject(CONFIG, json);
         this.configuration = new ConsumerConfiguration(jsonObject);
 
+        // both config and the base have a name field
+        JsonUtils.removeObject(json, CONFIG);
```

Comment:
this is done so regex won't get the wrong name field in case the config comes physically before the consumer name field.